corehq/apps/export/views.py

```diff
@@ -2454,7 +2455,16 @@ def download_daily_saved_export(req, domain, export_instance_id):
                 },
             )
     export_instance.last_accessed = datetime.utcnow()
-    export_instance.save()
+    try:
+        export_instance.save()
+    except ResourceConflict:
+        # if a user tries to download the same export multiple times in parallel
+        # there will be a conflict so retry with crit section
+        with CriticalSection(['export-last-accessed-{}'.format(export_instance_id)]):
```

Comment:
I thought that that may be a really long time to have a critical section for and overkill for every project except this one. Although I guess this makes the code less simple. i'll change it